pyspacemouse/pyspacemouse.py

```diff
@@ -357,6 +357,62 @@ def config_remove(self):
 # the IDs for the supported devices
 # Each ID maps a device name to a DeviceSpec object
 device_specs = {
+     "SpaceMouse Enterprise": DeviceSpec(
+        name="SpaceMouse Enterprise",
+        # vendor ID and product ID
+        hid_id=[0x256f, 0xc633],
+        led_id=[0x8, 0x4B],
+        mappings={
```

Comment:
In fact, I did not tinker with LED's ID, I do not know if SpaceMouse Enterprose has LEDs

Reply:
To be honest I didn't know what type of coordinate system is used for each manipulator. I copied this setting from other manipulator, coordinate system made sense to me, however I do not have any other already supported SpaceMouse to test how it is done there.

It would be great help if I knew what coordinate system should I use. I mean specification like:
- forward - backward X axis, where forward is positive etc.